README.md

```diff
@@ -6,11 +6,18 @@
 
 ## Introduction
 
-You will build a method, `square_array`, that squares each element in an array of numbers and returns a new array of these squared numbers.
+In this lab we will be building a method, `square_array`, that
+[squares](https://en.wikipedia.org/wiki/Square_(algebra)) each element in an
+array of numbers and returns a new array of these squared numbers.
+
+**Hint:** There is more than one way to square numbers! Google Ruby's exponent
+operator or refer back to the earlier lesson on simple math.
 
 ## Apply the `.each` Method to Iterate Over an Array
 
-Use an iterator and implement your own logic, *don't* use any built-in array methods other than `.each` (e.g. `.collect`, `.inject`).
+To build this method, use the iterator `.each` and implement your own logic. *Do
```

Comment:
remove use of .each

also include don't include exhaustive list of methods not to use do something like: "Use only while an Arrays, don't use the internet to find code you don't understand yet, etc."

Reply:
@sgharms had the same thought

Reply:
I meant let’s not include the exhaustive list of what not to use. In PAC we haven’t taught enumerables yet. So solutions and readers should t use them.